### ISSUE TITLE
Next topics

### ISSUE THREAD

**Issue by rburghol** [HARPgroup/vahydro#270]
- trends in winter base flow recharge over time
  - Model trends to detect base flow rechargee
- develop cbp model operational platform
  - compare original php with json fetcher results
- economic impacts modeling/analysis
- Demand projection for next 30-50 years, automating and standardizing for wsp
- Translate model 
- Conversion of VWP models
-

**Comment by durelles** [comment]
Thanks for sharing, great ideas to chew on/noodle. For the first one, I can't recall what the USGS scientist went with that? I'm guessing there's a paper/report available. Will think over coming days(hopefully not at 3am).

**Comment by rburghol** [comment]
Sam Austin out that pub together - I just added a link. Probably a good idea to add refs where available for all topics.

**Comment by jdkleiner** [comment]
Agreed, thanks for sharing Rob, I'll start adding/fleshing out potential topics!
